.github/workflows/run-examples.yaml

```diff
@@ -0,0 +1,71 @@
+name: Run Examples
+on:
+  workflow_call:
+    inputs:
+      sdk_version:
+        description: 'SDK version to run examples for'
+        required: false
+        type: string
+        default: 'LATEST'
+      jdk:
+        description: 'JDK version to use'
+        required: true
+        type: string
+        default: '17'
```

Comment:
Let's use the latest LTS version `21` as the default.

Reply:
Done.